scripts/helper.js

```diff
@@ -145,17 +145,21 @@ const GetDepTree = ( name ) => {
 
 
 /**
- * Get all folders within a given path
+ * Get all modules within a given path.
+ * Module is folder with package.json
  *
  * @param  {string}  thisPath - The path that contains the desired folders
  * @param  {boolean} verbose  - Verbose flag either undefined or true
  *
  * @return {array}            - An array of names of each folder
  */
-const GetFolders = ( thisPath, verbose ) => {
+const GetModules = ( thisPath, verbose ) => {
 	try {
 		let folders = Fs.readdirSync( thisPath ).filter(
-				thisFile => Fs.statSync(`${ thisPath }/${ thisFile }`).isDirectory()
+				thisFile => {
+					let path = `${ thisPath }/${ thisFile }`;
```

Comment:
```suggestion
					let path = Path.normalize( `${ thisPath }/${ thisFile }` );
```

We've had issues in the past with pathing across platforms, normalize should format this for us